feature/navigation/drawer/dropdown/src/main/kotlin/net/thunderbird/feature/navigation/drawer/dropdown/ui/folder/FolderListItem.kt

```diff
@@ -88,7 +89,7 @@ internal fun FolderListItem(
             if (displayChild === null) continue
             FolderListItem(
                 displayFolder = displayChild,
-                selected = false,
+                selected = selectedFolderId?.let { displayChild.id == selectedFolderId } ?: false,
```

Comment:
You can use equality check can be used instead of elvis for nullable boolean check:
```suggestion
                selected = selectedFolderId?.let { displayChild.id == selectedFolderId } == true,
```